src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx

```diff
@@ -17,29 +52,17 @@ export const OtherContributorsList = () => {
   }, []);
 
   const contributors = useMemo(() => {
-    const finalContributors = [];
     if (contributorsJson) {
-      // Add manually contributors which not inside git list
-      finalContributors.push(
-        <Link href="https://il.linkedin.com/in/rotem-dekel-7a8b12133" className={`${BASE_CLASS}_developer`}>
-          Rotem Dekel
-        </Link>
-      );
-
-      finalContributors.push(
-        <Link href="https://github.com/LeanyLabs" className={`${BASE_CLASS}_developer`}>
-          LeanyLabs
-        </Link>
-      );
-
       // developer contributors
       const developerContributors = contributorsJson
         .filter(contributor => !excludedDevelopers.has(contributor.id))
-        .map(contributor => (
-          <Link href={contributor.html_url} className={`${BASE_CLASS}_developer`} key={contributor.id}>
-            {contributor.login}
-          </Link>
-        ));
+        .map(contributor => ({ name: contributor.login, href: contributor.html_url, key: contributor.id }));
+
+      return STATIC_CONTRIBUTERS.concat(developerContributors).map(({ name, href, key }) => (
```

Comment:
I think we will want to display the static contributres even if we do not have contributorsJson. let's move the map logic to happend before the if(contributorsJson)